.eslintrc.json

```diff
@@ -1,9 +1,12 @@
 {
```

Comment:
You are not supposed to change this file ![see instruction](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)
  [see the correct content](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)